lib/tasks/interactive-new.js

```diff
@@ -81,7 +81,7 @@ class InteractiveNewTask extends Task {
         name: 'packageManager',
         type: 'list',
         message: 'Pick the package manager to use when installing dependencies:',
-        when: !newCommandOptions.yarn,
+        when: !newCommandOptions.packageManager,
```

Comment:
Take `--npm` and `--pnpm` into account as well for skipping this question.

Reply:
yaaas